.github/workflows/tests.yaml

```diff
@@ -29,3 +29,4 @@ jobs:
       - run: cargo run --example termination --target ${{ matrix.type.target }}
       - run: cargo clippy --all-targets --target ${{ matrix.type.target }} -- --deny warnings
       - run: cargo build --target ${{ matrix.type.target }}
+      - run: cargo run --example termination
```

Comment:
We already have this a couple of lines above. We should also move this to the system tests for consistency, but let's do this later in a later change.

Reply:
Done